src/pages/SketchSchemes.svelte

```diff
@@ -117,6 +118,7 @@
       {:else if $mode.mode == "split-route"}
         <SplitRouteMode />
       {/if}
+      <GeocoderLayer />
```

Comment:
It's kind of annoying to split the maplibre and sidebar components for this functionality that logically belongs together, but it's necessary because of how svelte-maplibre components need to be nested in the right place to work